### ISSUE TITLE
<TAGs> in code shouldn't be interpreted as HTML

### ISSUE THREAD

**Issue by mlopezgva** [obedm503/bootmark#14]
I was writing a short manual and needed to include some _vimScript_ code, with `<silent>` and some other code in `<` `>`. When showing up with bootmark, those strings disappeared. So, after some thought, I had to use htmlentities to show them. I think that, when inside _fenced code_, this shouldn't be necessary. #

**Comment by obedm503** [comment]
can you share a specific example? tags enclosed in backticks should be ignored instead of rendered

**Comment by mlopezgva** [comment]
Here's an example md:

```
### Test about parsing `<`  and `>`
This is a test about parsing text between "chevrons" (`<` and `>`) as HTML tags:

### Inline code
First test is about `some <tags> inside backticks`, or _inine code_;
First test is about `some &lt;tags&gt; inside backticks`, or _inine code_;

### Inside code block
Second test is about what seems like SGML tags inside a code block:

    vnoremap <silent> "$ :w !nc -q0 localhost 5900<CR>

With the `<` and `>` as _html entities_:

    vnoremap &lt;silent&gt; "$ :w !nc -q0 localhost 5900&lt;CR&gt;

Using ***bootmark*** I can't see the `silent` and `CR`.

```

Parsing this example using _bootmark_, I got this:

![image](https://cloud.githubusercontent.com/assets/996070/23610618/ec861b9e-0251-11e7-900c-245a61d8ac26.png)

**Comment by obedm503** [comment]
what version are you using? after some testing using the [showdown demo ](http://showdownjs.github.io/demo/#/%23%23%23%20Test%20about%20parsing%20%60%3C%60%20%20and%20%60%3E%60%0AThis%20is%20a%20test%20about%20parsing%20text%20between%20%22chevrons%22%20(%60%3C%60%20and%20%60%3E%60)%20as%20HTML%20tags:%0A%0A%23%23%23%20Inline%20code%0AFirst%20test%20is%20about%20%60some%20%3Ctags%3E%20inside%20backticks%60,%20or%20_inine%20code_;%0AFirst%20test%20is%20about%20%60some%20&lt;tags&gt;%20inside%20backticks%60,%20or%20_inine%20code_;%0A%0A%23%23%23%20Inside%20code%20block%0ASecond%20test%20is%20about%20what%20seems%20like%20SGML%20tags%20inside%20a%20code%20block:%0A%0A%20%20%20%20vnoremap%20%3Csilent%3E%20%22$%20:w%20!nc%20-q0%20localhost%205900%3CCR%3E%0A%0AWith%20the%20%60%3C%60%20and%20%60%3E%60%20as%20_html%20entities_:%0A%0A%20%20%20%20vnoremap%20&lt;silent&gt;%20%22$%20:w%20!nc%20-q0%20localhost%205900&lt;CR&gt;%0A%0AUsing%20***bootmark***%20I%20can't%20see%20the%20%60silent%60%20and%20%60CR%60.)and the [katex-latex demo](https://obedm503.github.io/katex-latex/) which uses bootmark. and they worked fine. 

I recommend you use a cdn like unpkg: https://unpkg.com/bootmark@0.7.1/dist/bootmark.bundle.min.js or jsdelivr: http://cdn.jsdelivr.net/jquery.bootmark/0.7.1/bootmark.bundle.min.js

**Comment by mlopezgva** [comment]
I cloned this repository in our development server (for internal documentation purposes).

I'll give a try to these links ASAP. Thank you.

**Comment by mlopezgva** [comment]
Changed the script included file and got this:

![image](https://cloud.githubusercontent.com/assets/996070/23625097/bcaf0798-0286-11e7-8a55-bf02b0d372b2.png)

HTML source:

![image](https://cloud.githubusercontent.com/assets/996070/23625127/daab7e66-0286-11e7-84ee-2f6afd1e5a0c.png)

Function parameters (title, MDtheme and codeTheme are strings):

```javascript
var BMrc = {
  css:'bootmark/dist/bootmark.min.css',
  html:{
    favicon: false,
    title: title,
    toc: (getParam('toc') === 'false') ? false : true,
    tocTitle: title,
    theme: MDtheme,
    prettifyTheme: codeTheme
  }
};
```

**Comment by obedm503** [comment]
confirmed. https://jsfiddle.net/dx0h3uqw/

what is interesting is that the demo's work fine. I'll look into it.

**Comment by obedm503** [comment]
it seems that `$.fn.text` is the onew causing problems. to retrive markdown, bootmark uses it and `.text()` ignores all html content in the element.

https://jsfiddle.net/f47x07gs/
 the fiddle hijacks `.text()` and replaces it with `.html()`

this should be addressed by 51937c0f63d4c41d30ec02c29c1ed78623f8fa3f and will be tested and merged into development
